.github/workflows/ci.yml

```diff
@@ -25,8 +25,13 @@ jobs:
   lingua-franca:
     runs-on: ubuntu-latest
     steps:
+      - name: Check out reactor-ts repository
+        uses: actions/checkout@v3
+        with: 
+          path: "ts-reactor"
+          fetch-depth: 0
       - name: Read lingua-franca-ref.txt
-        run: echo "lingua-franca-ref=$(wget -qO- https://raw.githubusercontent.com/lf-lang/reactor-ts/${{ github.head_ref || github.ref_name }}/lingua-franca-ref.txt)" >> $GITHUB_ENV
+        run: echo "lingua-franca-ref=$(cat ./reactor-ts/lingua-franca-ref.txt)" >> $GITHUB_ENV
       - name: Check out lingua-franca repository
```

Comment:
Don't we have to designate a `path` for this one as well? The root will already have `reactor-ts` in it, so it won't be empty.

Reply:
Hm, I guess it's OK, judging by the current run.

Reply:
Actually - no, you are right and this is a coincidence, let me change this a bit......